### DISCUSSION TITLE
Welcome to awesome-postgres Discussions!

### DISCUSSION THREAD

**Discussion by edib** [pg-tr/awesome-postgres#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hwpplayer1** [comment]
Hi I guess it will be fine to add a Postgres Code of Conduct Contributing and Postgres license and then issues template pull request template Authors files

I know it is hard to maintain by a single person So here i am to contribute.

Thanks.
